src/utils/crossPlatform.test.ts

```diff
@@ -68,5 +117,25 @@ describe('checking whether a link should be opened in a new tab', () => {
       const mouseEvent = new MouseEvent('click')
       expect(shouldOpenLinkInNewTab(mouseEvent)).toBeFalsy()
     })
+
+    it('triggers a copy when the ctrl key and c key are pressed', () => {
+      const keyEvent = new KeyboardEvent('keydown', {
+        ctrlKey: true,
+        key: 'KeyC',
+      })
+      expect(keyboardCopyTriggered(keyEvent)).toBeTruthy()
+    })
+
+    it('does not trigger a copy when the ctrl key and d key are pressed', () => {
+      const keyEvent = new KeyboardEvent('keydown', {
+        ctrlKey: true,
+        key: 'KeyD',
+      })
+      expect(keyboardCopyTriggered(keyEvent)).toBeFalsy()
+    })
+
+    it('gets the correct selection', () => {
+      expect(userSelection()).toBe('test')
+    })
```

Comment:
these test changes are solid, but they only target 1/4 the cases - when the modifier key is pressed but the `C` key isn't pressed. Can you also add cases for the other three cases?

- modifier key is pressed && `C` key is pressed
- modifier key is not pressed && `C` key is pressed
- modifier key is not pressed && `C` key is not pressed